two-factor.php

```diff
@@ -248,14 +248,14 @@ function wpcom_vip_two_factor_admin_notice() {
 			<div class="dashicons dashicons-warning" style="display:flex;float:left;margin-right:2rem;font-size:38px;align-items:center;margin-left:-20px;color:#ffb900;"></div>
 			<div>
 				<p style="font-weight:bold; font-size:16px;">
-					<a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">Two Factor Authentication</a> is required to edit content on this site.
+					Access requires your account to have <a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">Two-factor authentication</a> enabled.
```

Comment:
```suggestion
					Your account requires <a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">two-factor authentication</a> to be enabled.
```

Reply:
While we know the plugin is called Two Factor Authentication, the customer doesn't need to treat it as a proper noun.

Reply:
That's true, thanks for the suggestion